### ISSUE TITLE
Error with new Chrome V8 powered Apps Script runtime

### ISSUE THREAD

**Issue by Kocytean** [brucemcpherson/cGoa#2]
The word 'package' seemingly wasn't a reserved word in the previous Apps Script runtime? I have disabled the new one and it works as usual, in the newer one, Stackdriver logged the following error message on encountering it as a variable name:
`SyntaxError: Unexpected strict mode reserved word at [unknown function](GoaApp:43:34)`

Since newer users will get auto'd to the the new runtime, this should be fixed.

**Comment by brucemcpherson** [comment]
Good catch

thanks - I'll take a look at it.

**Comment by tourBermuda** [comment]
Hi 
I have updated my project to use the latest version of cGoa. However I am getting the following error. 
SyntaxError: Unexpected strict mode reserved word (line 43, file "GoaApp", project "cGoa")

**Comment by brucemcpherson** [comment]
Yes - I've updated to remove the word thats now reserved in v8 . However
there may be other issues to do with cross site redirection that I
havent looked into yet.

**Comment by tourBermuda** [comment]
Any idea if this a simple fix. if not, what other projects have a similar function. An help you can give will be appreciated. This has completely shutdown my project.

**Comment by brucemcpherson** [comment]
have you tried the version ive just updated?

**Comment by tourBermuda** [comment]
OK so I updated to 31. I notice a scope is missing from my project properties section.
"https://www.googleapis.com/auth/script.external_request"

Can you tell me what this means. 
I am getting an error that says :
The Scopes must be in a valid array (line 29, file "Code")
node of my files are named "Code"

**Comment by tourBermuda** [comment]
It's been a couple years since wrote this code. After some more inspection actually your script is include, but It does not look like I'm using it. back then I had trying to send data to a google firebase DB. I add cGoa and GSApp to get the the Oath connection to firebase. 

serverToken = new GSApp.init(key, ['https://www.googleapis.com/auth/firebase.database', 'https://www.googleapis.com/auth/userinfo.email'], clientEmail);

Sorry did not mean to waste your time.
could my error:
The Scopes must be in a valid array (line 29, file "Code")

be talking about these scope in the serverToken variable?

**Comment by zstatmanweil** [comment]
Hello, I am having the same issue "The Scopes must be in a valid array". My script has worked for years and the issue just started. I did just start getting the message about the new Chrome V8 runtime so wondering if that is related. @tourBermuda , did you ever figure out the issue?
